### PR TITLE
SO-3921: new duplicate refset member rule

### PR DESCRIPTION
**Pull request by molnarlaura**
These changes in rule 669 (former name: rule_duplicate_member) will support Language and Attribute value type refsets as well.
And a new rule (rule 670) was created for Association type refset duplicate members.

### CODE REVIEW

File: snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json [hunk 1]
```diff
@@ -72,12 +72,21 @@
     "implementation": "scripts/rule_mrcm_constraint_type.groovy"
   },
   {
-  	"id": "rule_duplicate_members",
+  	"id": "669",
     "toolingId": "com.b2international.snowowl.terminology.snomed",
     "messageTemplate": "Members must not duplicate each ohter.",
```
Comment (apeteri) [inline snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json]: ⚠️ Typo!

File: snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json [hunk 2]
```diff
@@ -72,12 +72,21 @@
...
     "severity": "ERROR",
     "checkType": "FAST",
     "type": "script-groovy",
-    "implementation": "scripts/rule_duplicate_members.groovy"
+    "implementation": "scripts/rule669.groovy"
+  },
+  {
+  	"id": "670",
+    "toolingId": "com.b2international.snowowl.terminology.snomed",
+    "messageTemplate": "Association type reference set members must not duplicate each ohter.",
```
Comment (apeteri) [inline snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json]: ⚠️ Also a typo.

**Review by apeteri — CHANGES_REQUESTED** [review]

File: snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json (hunk 1 above)
Comment (molnarlaura) [reply snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json]: ready

File: snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json (hunk 2 above)
Comment (molnarlaura) [reply snomed/com.b2international.snowowl.validation.snomed/src/main/resources/validation-rules-common.json]: ready

**Review by apeteri — APPROVED** [review]

**Pull request merged**